models/quality_measures/intermediate/cbe0101_falls_plan_of_care/quality_measures__int_cbe0101_denominator.sql

```diff
@@ -0,0 +1,271 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False)))) | as_bool
+   )
+}}
+
+with visit_codes as (
+
+    select
+          code
+        , code_system
+    from {{ ref('quality_measures__value_sets') }}
+    where lower(concept_name) in (
+          'annual wellness visit'
+        , 'audiology visit'
+        , 'home healthcare services'
+        , 'nursing facility visit'
+        , 'occupational therapy evaluation'
+        , 'office visit'
+        , 'physical therapy evaluation'
+        , 'online assessments'  -- for clinical data sources
+        , 'telephone visits'    -- for clinical data sources
```

Comment:
Can you add these additional concepts? They are from the eCQM (clinical) version of this measure.
```
   'Care Services in Long Term Residential Facility'
 , 'Discharge Services Nursing Facility'
 , 'Encounter Inpatient'
 , 'Ophthalmological Services'
 , '"Preventive Care Services Established Office Visit, 18 and Up"'
 , 'Preventive Care Services Individual Counseling'
 , '"Preventive Care Services Initial Office Visit, 18 and Up"'
 ```